Tools/autotest/sim_vehicle.py

```diff
@@ -678,8 +678,10 @@ def start_vehicle(binary, opts, stuff, spawns=None):
         c = ["-I" + str(i)]
         if spawns is not None:
             c.extend(["--home", spawns[i]])
+        if opts.auto_sysid and opts.sysid is None and 0 <= i <= 254:
+            c.extend(["--sysid", str(i + 1)])
         os.chdir(i_dir)
-        run_in_terminal_window(cmd_name, cmd + c)
+        run_in_terminal_window(cmd_name, c + cmd)
```

Comment:
this seems to break basic sim_vehicle.py usage

Reply:
Yea, sorry, I didn't realize cmd[0] was the command name (I thought it was cmd_name). I just dropped that commit.